src/third/redux-persist/getStoredState.js

```diff
@@ -1,98 +1,93 @@
+  const restoredState = {};
   let completionCount = 0;
 
   (async () => {
     let err = null;
     let allKeys = undefined;
     try {
-      allKeys = await storage.getAllKeys()
+      allKeys = await storage.getAllKeys();
     } catch (e) {
-      err = e
+      err = e;
     }
 
     if (err) {
-      logging.warn(err, { message: 'redux-persist/getStoredState: Error in storage.getAllKeys' })
-      complete(err)
-      return
+      logging.warn(err, { message: 'redux-persist/getStoredState: Error in storage.getAllKeys' });
+      onComplete(err);
+      return;
     }
 
-    let persistKeys = allKeys.filter((key) => key.indexOf(keyPrefix) === 0).map((key) => key.slice(keyPrefix.length))
-    let keysToRestore = persistKeys.filter(passWhitelistBlacklist)
+    const persistKeys = allKeys
+      .filter(key => key.indexOf(keyPrefix) === 0)
+      .map(key => key.slice(keyPrefix.length));
+    const keysToRestore = persistKeys.filter(key => whitelist.indexOf(key) !== -1);
 
-    let restoreCount = keysToRestore.length
-    if (restoreCount === 0) complete(null, restoredState)
-    keysToRestore.forEach((key) => {
+    const restoreCount = keysToRestore.length;
+    if (restoreCount === 0) {
+      onComplete(null, restoredState);
+    }
+    keysToRestore.forEach(key => {
       (async () => {
         let err = null;
         let serialized = null;
         try {
-          serialized = await storage.getItem(createStorageKey(key))
+          serialized = await storage.getItem(createStorageKey(key));
         } catch (e) {
-          err = e
+          err = e;
+        }
+        if (err) {
+          logging.warn(err, {
+            message: 'redux-persist/getStoredState: Error restoring data for a key.',
+            key,
+          });
+        } else {
+          restoredState[key] = rehydrate(key, serialized);
+        }
+        completionCount += 1;
+        if (completionCount === restoreCount) {
+          onComplete(null, restoredState);
         }
-        if (err) logging.warn(err, { message: 'redux-persist/getStoredState: Error restoring data for a key.', key})
-        else restoredState[key] = rehydrate(key, serialized)
-        completionCount += 1
-        if (completionCount === restoreCount) complete(null, restoredState)
-      })()
-    })
+      })();
+    });
   })();
 
-  function rehydrate (key, serialized) {
-    let state = null
+  function rehydrate(key, serialized) {
+    let state = null;
 
     try {
-      let data = deserializer(serialized)
-      state = data
+      const data = deserializer(serialized);
+      state = data;
     } catch (err) {
-      logging.warn(err, { message: 'redux-persist/getStoredState: Error restoring data for a key.', key })
+      logging.warn(err, {
+        message: 'redux-persist/getStoredState: Error restoring data for a key.',
+        key,
+      });
     }
 
-    return state
-  }
-
-  function complete (err, restoredState) {
-    onComplete(err, restoredState)
-  }
```

Comment:
getStoredState [nfc]: Inline `complete`.

Huh. I am mystified as to why this indirection was here in the first place. Even looking at the very first version we vendored (with `git log --stat -p --reverse src/third/redux-persist/getStoredState.js`), it seems like it doesn't have any effect and the code would be improved by this inlining.

Perhaps it made more sense either in an earlier version of redux-persist (before the one we vendored from), or in a contemplated future API for it that hadn't been implemented.

In any case, I think this inlining makes just as much sense with the preceding require-`onComplete` change dropped as it does with that change included.